modules/mobile-agents/android/client/src/org/wso2/mdm/agent/api/Root.java

```diff
@@ -42,7 +47,6 @@ public boolean isDeviceRooted() {
 		if (checkRootByBuildTags()) {
 			return true;
 		}
-
 		return false;
```

Comment:
May be we can use "return checkRootByBuildTags();" directly without writing an if condition here.